packages/x-components/src/x-modules/next-queries/components/next-query-preview.vue

```diff
@@ -209,4 +224,56 @@ In this example, the ID of the results will be rendered along with the name.
   };
 </script>
 ```
+
+### Play with props
+
+In this example, the suggestions has been limited to render a maximum of 4 items.
+
+```vue
+<template>
+  <NextQueryPreview :suggestion="suggestion" #result="{ result }">
+    <BaseGrid
+      #default="{ item }"
+      :animation="gridAnimation"
+      :columns="maxItemsToRender"
+      class="x-padding--00"
+    >
+      <BaseResultLink :result="result">
+        <BaseResultImage :result="result" class="x-result__picture" />
```

Comment:
Shouldn't this be something as follows?

```suggestion
  <NextQueryPreview :suggestion="suggestion" :maxItemsToRender="4" #default="{ results }">
    <BaseGrid #default="{ item }" :animation="gridAnimation" :items="results" class="x-padding--00">
      <BaseResultLink :result="item">
        <BaseResultImage :result="item" class="x-result__picture" />
```